### PR TITLE
test: use locally published version in E2E project setup

### PR DESCRIPTION
**Pull request by clydin**
This removes the need to perform a second _special_ local build of the packages during the E2E test process.

This also has the benefit of causing CI to fail if the package versions in `latest-versions.ts` are not kept synchronized with the CLI package versions.

### CODE REVIEW

File: tests/legacy-cli/verdaccio.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ packages:
     access: $all
     publish: $all
 
-  '@ngtools/*':
+  '@ngtools/webpack':
```
Comment (filipesilva) [inline tests/legacy-cli/verdaccio.yaml]: Was this change necessary or was it just cleanup? Just wondering, because `@angular-devkit/*` still has the star above.

Comment (clydin) [reply tests/legacy-cli/verdaccio.yaml]: “@ngtools/json-schema” is used by ng-packgr.  Since it no longer exists, it’s not published locally which causes the installation of ng-packgr to fail within the E2E tests.
